ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue

```diff
@@ -9,74 +9,8 @@
-          <v-col cols="12">
-            <label class="generic-label">
-              Role
-            </label>
-          </v-col>
-          <v-col class="pt-2 pb-9">
-            <v-radio-group
-              id="owner-role-options"
-              class="mt-0 pr-2" row
-              hide-details="true"
-              v-model="owner.partyType"
-            >
-              <v-radio
-                id="owner-option"
-                class="owner-radio pr-4"
-                label="Owner"
-                active-class="selected-radio"
-                :disabled="isTransferToExecutorProbateWill ||
-                  isTransferToExecutorUnder25Will ||
-                  isTransferToAdminNoWill"
-                v-model="HomeOwnerPartyTypes.OWNER_IND"
-              />
-              <v-tooltip
-                v-if="isTransferDueToDeath || isFrozenMhr"
-                top
-                nudge-right="18"
-                content-class="top-tooltip pa-5"
-                transition="fade-transition"
-              >
-                <template v-slot:activator="{ on }">
-                  <v-radio
-                    v-on="on"
-                    id="executor-option"
-                    class="executor-radio px-4"
-                    active-class="selected-radio"
-                    :disabled="disableNameFields || isTransferToAdminNoWill || isFrozenMhr"
-                    v-model="HomeOwnerPartyTypes.EXECUTOR"
-                   >
-                    <template v-slot:label><div :class="{'underline' : !disableNameFields}">Executor</div></template>
-                  </v-radio>
-                </template>
-                  An Executor is the personal representative named in the will or appointed by court to carry out the
-                  requirements of a will. They are also referred to as trustee or liquidator (in Quebec).
-              </v-tooltip>
-              <v-radio
-                id="trustee-option"
-                class="trustee-radio px-4"
-                label="Trustee"
-                active-class="selected-radio"
-                :disabled="isTransferDueToDeath || isTransferToExecutorProbateWill || isFrozenMhr"
-                v-model="HomeOwnerPartyTypes.TRUSTEE"
-              />
-              <v-radio
-                id="administrator-option"
-                class="administrator-radio pl-4"
-                label="Administrator"
-                active-class="selected-radio"
-                :disabled="isTransferToSurvivingJointTenant ||
-                  isTransferToExecutorUnder25Will ||
-                  isTransferToExecutorProbateWill ||
-                  isFrozenMhr"
-                v-model="HomeOwnerPartyTypes.ADMINISTRATOR"
-              />
-            </v-radio-group>
-          </v-col>
-        </v-row>
+        <!-- Owner Roles Component-->
+        <HomeOwnerRoles :partyType="owner.partyType"  @update:partyType="owner.partyType = $event"/>
```

Comment:
What if we update the `owner.partyType` in the child component without emitting? It is still reactive, right. Or you wanted to follow the pattern of emitting events up?

Reply:
You bet, it's reactive both ways. 
I don't believe you could update it in the child without it emitting. Is there a specific scenario you are thinking of?